cli/compose/schema/schema_test.go

```diff
@@ -50,3 +50,27 @@ func TestValidateInvalidVersion(t *testing.T) {
 	assert.Error(t, err)
 	assert.Contains(t, err.Error(), "unsupported Compose file version: 2.1")
 }
+
+type array []interface{}
+
+func TestValidatePlacement(t *testing.T) {
+	config := dict{
+		"version": "3.3",
+		"services": dict{
+			"foo": dict{
+				"image": "busybox",
+				"deploy": dict{
+					"placement": dict{
+						"preferences": array{
+							dict{
+								"spread": "node.labels.aliyun.zone",
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	assert.NoError(t, Validate(config, "3.3"))
```

Comment:
This test would be more appropriate as a test in `cli/compose/loader/`. There is a TestFull that you can add to

The tests in this package are for the schema loading and validation.